NeoML/include/NeoML/Dnn/Layers/MobileNetV2BlockLayer.h

```diff
@@ -0,0 +1,108 @@
+/* Copyright © 2017-2022 ABBYY Production LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+--------------------------------------------------------------------------------------------------------------*/
+
+#pragma once
+
+#include <NeoML/NeoMLDefs.h>
+#include <NeoML/Dnn/Dnn.h>
+
+namespace NeoML {
+
+// This layer computes a block in a MobileNetV2 architecture
+//
+// The block may be without residual connection
+//     conv1x1 (expand) -> relu (expandReLU) -> channelwiseConv3x3 -> relu (channelwiseReLU) -> conv1x1 (down)
+// or it may be with residual connection
+//     -+--> block without residual ----> sum ->
+//      |                                  |
+//      +----------------------------------+
+//
+// This layer is faster and consumes less memory than the composite of layers but it has some restrictions:
+//     - this layer is untrainable
+//     - all 1x1 convolutions must have no paddings and stride == 1
+//     - channelwise convolution must have stride 1 or 2, padding == 1 and dilation == 1
+//     - only ReLU activation is supported (upper thresholds in ReLU are supported)
+//     - free terms are supported (but may be nullptr) for all the convolutions i nblock
+class NEOML_API CMobileNetV2BlockLayer : public CBaseLayer {
+	NEOML_DNN_LAYER( CMobileNetV2BlockLayer )
+public:
+	explicit CMobileNetV2BlockLayer( IMathEngine& mathEngine );
+	~CMobileNetV2BlockLayer();
+
+	// Expand convolution and RelU parameters
```

Comment:
Are we needed in such methods for non-trainable layers? Let's make the interface of the class smaller!

Reply:
Done.